core-plugins/docs/TPFSAvro-batchsink.md

```diff
@@ -31,6 +31,17 @@ Defaults to formatting partitions such as ``2015-01-01/20-42.142017372000``.
 **timeZone:** The string ID for the time zone to format the date in. Defaults to using UTC.
 This setting is only used if ``filePathFormat`` is not null.
 
+**partitionOffset:** Amount of time to subtract from the pipeline runtime to get the output partition. Defaults to 0m.
```

Comment:
"Amount of time to subtract from the pipeline runtime to determine the output partition. Defaults to 0m. "